devel/hivex/Portfile

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           perl5 1.0
+
+name                hivex
+version             1.3.23
+revision            0
+categories          devel
+license             LGPL-2.1
+maintainers         {outlook.com:mohd.akram @mohd-akram} openmaintainer
+
+description         A self-contained library for reading and writing Windows \
+                    Registry \"hive\" binary files.
+
+long_description    {*}${description}
+
+homepage            https://github.com/libguestfs/hivex
+
+master_sites        https://download.libguestfs.org/hivex/
+
+checksums           rmd160  ed4041d779dd4d31ebab7d42f85fb6a59aadc84e \
+                    sha256  40cf5484f15c94672259fb3b99a90bef6f390e63f37a52a1c06808a2016a6bbd \
+                    size    738076
+
+perl5.default_branch  5.34
+perl5.branches        ${perl5.default_branch}
+perl5.create_variants ${perl5.branches}
+
+depends_build-append port:pkgconfig
```

Comment:
this doesn't follow the "four-spaces-indentation-rule"

Reply:
Thanks, fixed.